docs/en/trust.rst

```diff
@@ -120,13 +118,13 @@ Below a non-normative example of Trust Anchor Entity Configuration, where each p
 Entity Configuration and Entity Statement
 `````````````````````````````````````````
 
-The Entity Configuration is the federation metadata that an Entity publishes about itself, that is verifiable with a trusted third party. The Entity Configuration is signed and it is verifiable with one of the public key contained in it and also in the Entity Statement issued by the Trust Anchor or its Intermediate, defined in a parameter called authority_hints. The Entity Configuration may also contain one or more Trust Marks regarding its issuer.
+The Entity Configuration is the federation metadata that an Entity publishes about itself, which is verifiable by a trusted third party. The Entity Configuration is signed, and it can be verified with one of the public keys contained within it, as well as within the Entity Statement issued by the Trust Anchor or its Intermediate. This is defined in a parameter called authority_hints. The Entity Configuration may also contain one or more Trust Marks regarding its issuer.
```

Comment:
Here "by" seems that the verifier is the TTP, while we want to say that it is verifiable thanks to a TTP :-)

Reply:
```suggestion
The Entity Configuration is the federation metadata that an Entity publishes about itself, which is verifiable thanks to a trusted third party. The Entity Configuration is signed, and it can be verified with one of the public keys contained within it, as well as within the Entity Statement issued by the Trust Anchor or its Intermediate. This is defined in a parameter called authority_hints. The Entity Configuration may also contain one or more Trust Marks regarding its issuer.
```